src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  console.log(todos);
+
+  return (
+    <>
+      {todos.map((todo: Todo) => (
+        <div key={todo.id} data-cy="Todo" className="todo completed">
```

Comment:
completed class should be only for completed todos.
it is a reason of current failed test.
Compare demo link with working example.
![Screenshot 2025-06-09 at 15 46 03](https://github.com/user-attachments/assets/b8060e77-9b7f-4688-ab1b-e4a2e3bfb981)
![Screenshot 2025-06-09 at 15 49 04](https://github.com/user-attachments/assets/402a77e6-6d87-42f1-8ce1-0647cc115ceb)
![Screenshot 2025-06-09 at 15 50 10](https://github.com/user-attachments/assets/e669af46-a4f7-4b14-a4bf-397c709410c8)